@hts-forking/src/index.js

```diff
@@ -168,6 +168,22 @@ async function getHtsStorageAt(address, requestedSlot, blockNumber, mirrorNodeCl
         );
     }
 
+    // Encoded `address(tokenId).isAssociated()` slot
+    // slot(256) = `isAssociated`selector(32) + padding(192) + accountId(32)
+    if (nrequestedSlot >> 224n === 0x4d8fdd6dn) {
```

Comment:
Is there a test that confirms this mapping for `0x4d8fdd6dn` works and ensures no regression.
I don't see one in the PR but let me know if it already exists.

Reply:
Yes, it is tested through the `test_IHRC719_with_real_accounts()` in `IHRC719.t.sol`

Reply:
Two things here

**1.** I'd add a unit test that covers this case. You can include it in `getHtsStorageAt.test.js`. There you'll find similar cases for balances and allowances in there.
**2.** Also please include a validation right after the `if` to ensure that the padding is zero. Given the association signature, the padding should be the same as `balanceOf`. Otherwise return `null`.

Reply:
1. Done, added unit tests both for MFCT and USDC tokens with the following test cases:
```javascript
{
    name: 'token relationship is found',
    fn: (/** @type {string} */ idOrAliasOrEvmAddress, /** @type {string} */ _tid) =>
        require(`./data/${symbol}/getTokenRelationship_${idOrAliasOrEvmAddress}`),
},
{
    name: 'token relationship is empty',
    fn: (/** @type {string} */ _accountId, /** @type {string} */ _tid) => ({ tokens: [] }),
},
{
  name: 'token relationship is null',
  fn: (/** @type {string} */ _accountId, /** @type {string} */ _tid) => null,
},
```
2. Changed it to account for padding inside the if itself (similar to how it is done for the `balanceOf` selector:
```javascript
    if (
        nrequestedSlot >> 32n ===
        0x4d8fdd6d_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000n
    ) {
      ...
    }
```